index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header id="header" >
+        <span id="logo" >Robertson Arthur</span>
+        <button type="button" class="btn header-btn">
+            <img src="./icons/Icon-Menumenu-icon.png" alt="Menu-icon">
+        </button>
```

Comment:
The hamburger button represents the navigation;

- [ ] In order to use semantic tags, kindly, wrap the button inside a `nav` tag;

Reply:
Done

Reply:
Thank you very much for a well detailed review